composer.json

```diff
@@ -15,40 +20,40 @@
         "symfony/process": "^4.4|^5.4"
     },
     "require-dev": {
-        "behat/behat": "^3.7",
-        "behat/mink-selenium2-driver": "^1.4",
-        "dmore/behat-chrome-extension": "^1.3",
-        "dmore/chrome-mink-driver": "^2.7",
-        "friends-of-behat/mink": "^1.8",
-        "friends-of-behat/mink-browserkit-driver": "^1.4",
-        "friends-of-behat/mink-debug-extension": "^2.0",
-        "friends-of-behat/mink-extension": "^2.4",
-        "friends-of-behat/page-object-extension": "^0.3",
-        "friends-of-behat/suite-settings-extension": "^1.0",
-        "friends-of-behat/symfony-extension": "^2.1",
-        "friends-of-behat/variadic-extension": "^1.3",
-        "friendsoftwig/twigcs": "v6.0.0",
-        "j13k/yaml-lint": "^1.1",
-        "php-parallel-lint/php-parallel-lint": "^1.0",
+        "behat/behat": "3.10.0",
+        "behat/mink-selenium2-driver": "1.6.0",
+        "dmore/behat-chrome-extension": "1.4.0",
+        "dmore/chrome-mink-driver": "2.8.1",
+        "friends-of-behat/mink": "1.10.0",
+        "friends-of-behat/mink-browserkit-driver": "1.6.1",
+        "friends-of-behat/mink-debug-extension": "2.1.0",
+        "friends-of-behat/mink-extension": "2.7.1",
+        "friends-of-behat/page-object-extension": "0.3.2",
+        "friends-of-behat/suite-settings-extension": "1.1.0",
+        "friends-of-behat/symfony-extension": "2.3.1",
+        "friends-of-behat/variadic-extension": "1.5.0",
+        "friendsoftwig/twigcs": "6.0.0",
+        "j13k/yaml-lint": "1.1.4",
+        "php-parallel-lint/php-parallel-lint": "1.3.2",
         "phpmd/phpmd": "2.12.0",
         "phpro/grumphp": "1.12.0",
         "phpspec/phpspec": "7.2.0",
-        "phpstan/extension-installer": "^1.0",
+        "phpstan/extension-installer": "1.1.0",
         "phpstan/phpstan": "1.7.14",
         "phpstan/phpstan-doctrine": "1.3.9",
         "phpstan/phpstan-strict-rules": "1.2.3",
         "phpstan/phpstan-webmozart-assert": "1.2.0",
-        "sylius-labs/coding-standard": "v4.1.1",
-        "phpunit/phpunit": "^9.0",
-        "sebastian/phpcpd": "^6.0",
-        "seld/jsonlint": "^1.7",
+        "sylius-labs/coding-standard": "4.1.1",
+        "phpunit/phpunit": "9.5.21",
+        "sebastian/phpcpd": "6.0.3",
+        "seld/jsonlint": "1.9.0",
         "slevomat/coding-standard": "7.2.1",
-        "symfony/browser-kit": "^5.1",
-        "symfony/debug-bundle": "^5.1",
-        "symfony/dotenv": "^5.1",
-        "symfony/intl": "^5.1",
-        "symfony/web-profiler-bundle": "^5.1",
-        "webmozart/assert": "^1.8"
+        "symfony/browser-kit": "5.4.3",
+        "symfony/debug-bundle": "5.4.3",
+        "symfony/dotenv": "5.4.5",
+        "symfony/intl": "5.4.10",
+        "symfony/web-profiler-bundle": "5.4.10",
+        "webmozart/assert": "1.11.0"
```

Comment:
:thinking: not directly related, but as this dependency is used in source, it should be in `require`, not in `require-dev`

Reply:
You're absolutely right, I'm changing that!
Thanks